src/components/CountDown.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+class CountDown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: ''
+    }
+  }
+
+  componentDidMount() {
+    this.renderTime()
+    this.timer = setInterval(() => this.renderTime(), 1000)
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  renderTime() {
+    if (this.props.how_long) {
+      const { startedAt } = this.props;
+      const left = Number(startedAt) + 24 * 60 * 60 * 1000 - Date.now();
+      this.calculationTime(left)
+    } else {
+      const left = 24 * 60 * 60 * 1000 - Date.now() % (24 * 60 * 60 * 1000);
+      this.calculationTime(left)
```

Comment:
you call calculationTime in both cases, how about only setting "left" in if and then call calculationTime?